website/content/docs/fork-workflow.md

```diff
@@ -26,18 +28,18 @@ When using the GitHub backend, you can use Netlify CMS to accept contributions f
 
 ## Using the Fork Workflow
 
-When a user logs into the CMS who doesn't have write access to your repo, the CMS will ask them if it's all right to create a fork of your repo (or use their existing fork, if they already have one). They are then presented with the normal CMS interface. The published content shown is from the original repo, so it stays up-to-date as changes are made.
+When a user logs into Netlify CMS who doesn't have write access to your repo, the CMS will ask for permission to create a fork of your repo (or use their existing fork, if they already have one). They are then presented with the normal CMS interface. The published content shown is from the original repo, so it stays up-to-date as changes are made.
```

Comment:
In places where it doesn't feel too awkward, I try to use "Netlify CMS" instead of "the CMS".